e2e/main_test.go

```diff
@@ -0,0 +1,104 @@
+package e2e
+
+import (
+	"context"
+	"flag"
+	"log"
+	"os"
+	"testing"
+
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/e2e-framework/pkg/env"
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/envfuncs"
+	"sigs.k8s.io/e2e-framework/support/kind"
+)
+
+var (
+	testenv     env.Environment
+	agentImage  = flag.String("agent-image", "", "The proxy agent's docker image.")
+	serverImage = flag.String("server-image", "", "The proxy server's docker image.")
+)
```

Comment:
not a blocking change but why not put it together in a config struct ?

Reply:
How do you mean? I did it this way so it could be passed in easily from the command line (e.g. by the `Makefile`).